src/tests/test_transform.py

```diff
@@ -70,8 +70,17 @@ def test_beta_stable(self):
 			V_2020_2: {"beta": {betaAddon.addonId: betaAddon}, "stable": {stableAddon.addonId: stableAddon}},
 		})
 
-	def test_onlyNewerUsed(self):
-		"""Ensure only the newest addon is used for a version+channel"""
+	def test_isAddonNewer(self):
+		"""
+		Ensure only the newest addon is used for a version+channel.
+
+		getLatestAddons checks addons in order, and we want to ensure
+		the addon dictionary is updated to maintain the newest addon
+		for a version+channel.
+
+		This tests _isAddonNewer by confirming that the order of addons being added
+		does not effect the results.
```

Comment:
> the order of addons being added

This doesn't fit the abstraction, it requires knowledge of the internals of `getLatestAddons`. For someone without that familiarity, it sounds like "added to the addon-store-submissions" but that doesn't make sense here either.

Instead, I'd suggest talking about it in terms of: "the order of addons in the list (`addons` arg) does not affect the output."